test/src/test/java/org/corfudb/runtime/LogReplicationClientTest.java

```diff
@@ -0,0 +1,296 @@
+package org.corfudb.runtime;
+
+import com.google.common.collect.Iterables;
+import org.corfudb.integration.AbstractIT;
+import org.corfudb.runtime.LogReplication.LRClientId;
+import org.corfudb.runtime.LogReplication.LRClientInfo;
+import org.corfudb.runtime.LogReplication.ClientInfoKey;
+import org.corfudb.runtime.LogReplication.SinksInfoVal;
+import org.corfudb.runtime.collections.CorfuStore;
+import org.corfudb.runtime.collections.CorfuStoreEntry;
+import org.corfudb.runtime.collections.Table;
+import org.corfudb.runtime.collections.TableOptions;
+import org.corfudb.test.SampleSchema;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.stream.Stream;
+
+public class LogReplicationClientTest extends AbstractIT {
+    private static String corfuSingleNodeHost;
+
+    private static int corfuStringNodePort;
+
+    private static String singleNodeEndpoint;
+
+    /* A helper method that takes host and port specification, start a single server and
+     *  returns a process. */
+    private void runSinglePersistentServer(String host, int port) throws IOException {
+        new CorfuServerRunner()
+                .setHost(host)
+                .setPort(port)
+                .setLogPath(getCorfuServerLogPath(host, port))
+                .setSingle(true)
+                .runServer();
+    }
+
+    /**
+     * Load properties for a single node corfu server before each test
+     */
+    @Before
+    public void loadProperties() {
+        corfuSingleNodeHost = PROPERTIES.getProperty("corfuSingleNodeHost");
+        corfuStringNodePort = Integer.valueOf(PROPERTIES.getProperty("corfuSingleNodePort"));
+        singleNodeEndpoint = String.format("%s:%d",
+                corfuSingleNodeHost,
+                corfuStringNodePort);
+    }
+
+    /**
+     * Test registering replication client
+     *
+     * @throws Exception exception
+     */
+    @Test
+    public void testRegisterReplicationClient() throws Exception {
+        final String namespace = "CorfuSystem";
+        final String tableName = "LogReplicationRegistrationTable";
+        final String clientName = "client";
+
+        runSinglePersistentServer(corfuSingleNodeHost, corfuStringNodePort);
+        CorfuRuntime runtime = createRuntime(singleNodeEndpoint);
+        CorfuStore store = new CorfuStore(runtime);
```

Comment:
The lines till #76 is common for all tests. So we can place them in @Before method